### PR TITLE
dropdown-menu / handle no header

### PR DESCRIPTION
**Pull request by pvilalta**
hello team,

This PR brings a different way to use the the dropdown menu
In order to apply the design below, I made the dropdown to whether or no accept a "header" slot 
So from now on, when the dropdown is called without header, the menu will be displayed straightaway

I also added a method that align children menu dynamically to it parent

Screenshot 2022-10-19 at 15 33 10
Screenshot 2022-10-19 at 15 33 17

### CODE REVIEW

**Comment by Amoki** [comment]
Can you update the website to be able to play with the new props?

File: src/BIMDataComponents/BIMDataDropdownMenu/BIMDataDropdownMenu.vue [hunk 1]
```diff
@@ -79,7 +90,7 @@ export default {
       type: String,
       default: "down",
       validator: directionClass =>
-        ["down", "up", "right", "left"].includes(directionClass),
+        ["down", "up", "right", "left", "none"].includes(directionClass),
```
Comment (NicolasRichel) [inline src/BIMDataComponents/BIMDataDropdownMenu/BIMDataDropdownMenu.vue:93]: What's the purpose of the `"none"` value ?
I can't find where you use it..

**Comment by pvilalta** [comment]
the way I display the `:menuItems` props in the doc could be optimised, feedbacks are welcome

File: src/BIMDataComponents/BIMDataDropdownMenu/BIMDataDropdownMenu.vue [hunk 2]
```diff
@@ -110,10 +127,45 @@ export default {
...
   },
+  mounted() {
+    this.mutationObserver = new MutationObserver(this.isHeaderDisplayed);
+    this.mutationObserver.observe(this.$refs.dropdown, {
+      childList: true,
+      subtree: true,
+    });
+  },
   methods: {
+    isHeaderDisplayed() {
```
Comment (Amoki) [inline src/BIMDataComponents/BIMDataDropdownMenu/BIMDataDropdownMenu.vue:142]: We usually expect a method named `isSomething` to return a boolean and not to set a value somewhere else.

**Review by Kurtil — APPROVED** [review]

**Pull request merged**